front/pages/w/[wId]/a.tsx

```diff
@@ -0,0 +1,36 @@
+import { withDefaultUserAuthRequirements } from "@app/lib/iam/session";
+import { VaultResource } from "@app/lib/resources/vault_resource";
+
+// This page is temporarily kept to redirect users to the global vault.
+// It should be removed once the legacy non-vault UI is removed.
```

Comment:
can't we replace all remaining links to this page with `getDustAppsListUrl` ( and update getDustAppsListUrl to point to `/vaults/${vault.sId}/apps` instead of `/a`) ?

Reply:
This feels simpler since we're just going to delete that code no ?